src/database/index.tsx

```diff
@@ -41,6 +42,9 @@ const scenariosCollectionId = "scenarios";
 const facilitiesCollectionId = "facilities";
 const modelVersionCollectionId = "modelVersions";
 const usersCollectionId = "users";
+// TODO (#521): when the datasource stabilizes, change this to the real collection
+const shadowDataCollectionId = "reference_facilities_test";
+const shadowDataCovidCasesCollectionId = "covidCases";
```

Comment:
Annoying Nit:  Do we want to remove references to "shadow" in the codebase?  It seems to me this term may lose its meaning over time.

Also, "Data" here is probably too generic since as far as I know this will always be referring to facilities.

Soooo... because I'm boring, I'd suggest updating these `consts` and the other remaining functions, types, etc.  to "Reference Facility"

`referenceFacilitiesCollectionId`
`referenceFacilitiesCovidCasesCollectionId`
`getReferenceFacilities`
`ReferenceFacility`
`ReferenceCovidCase`

etc, etc...

Reply:
If this feels like too much work or if we feel like "shadow" is self-explanatory, feel free to ignore this comment.

Reply:
I agree with this, actually. Reference facility seems like a wiser long-term naming choice in the actual code.

Reply:
I was on the fence about this but I am definitely sympathetic to this argument .... with you weighing in now too I am inclined to just change it so that we don't regret the fact that it has a weird name later